export/distro/registrations.go

```diff
@@ -88,6 +88,9 @@ func (reg *registrationInfo) update(newReg export.Registration) bool {
 		// TODO reg.sender = distro.NewAzureSender("TODO URL")
 	case export.DestRest:
 		reg.sender = NewHTTPSender(newReg.Addressable)
+	case export.DestXMPP:
+		reg.sender = NewXmppSender(newReg.Addressable)
```

Comment:
Use `NewXMPPSender` to follow Go standard on initialisms: https://github.com/golang/go/wiki/CodeReviewComments#initialisms